packages/web/src/components/createCell.tsx

```diff
@@ -1,16 +1,48 @@
+import { useMemo } from 'react'
+
 import type { DocumentNode } from 'graphql'
 import type { A } from 'ts-toolbelt'
 
 import { useQuery } from './GraphQLHooksProvider'
 
+enum Status {
+  Error = 'ERROR',
+  Loading = 'LOADING',
+  Success = 'SUCCESS',
+  Updating = 'UPDATING',
+  Empty = 'EMPTY',
+  NullDataError = 'NULL_DATA_ERROR',
+}
```

Comment:
We prefer to not use enums since they emit code: https://fettblog.eu/tidy-typescript-avoid-enums/

Reply:
You can just use a union type with strings, and that'll give you the same kind of safety but without having to emit code:

`type status = "ERROR"  | "LOADING" | ...`